<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGradesForStudent</class>
 <widget class="QDialog" name="AddGradesForStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="Student_Name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="grade">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Course_Name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student_Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select course</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Grade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Grade">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_course">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Student_Id">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student_ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
